packages/logger/src/LoggerUtils.ts

```diff
@@ -112,14 +113,19 @@ export class LoggerUtils {
      * @returns True - if the given property is to be treated as a special value; False - otherwise
      */
     public static isSpecialValue = (prop: string): boolean => {
-        // Others: token, job_load, job_pmahlq
-        const specialValues = ["user", "password", "tokenValue", "keyPassphrase"];
-        // TODO: add special values based on secure properties in:
-        // - meta (imperative config object from plugins)
-        // - schema.json
+        let specialValues = ["user", "password", "tokenValue", "keyPassphrase"];
 
-        for (const profile of LoggerUtils.profiles) {
+        const getPropertyNames = (prop: ICommandProfileProperty): string[] => {
+            const ret: string[] = [];
+            ret.push(prop.optionDefinition?.name);
+            prop.optionDefinitions?.map(opDef => ret.push(opDef.name));
+            return ret;
```

Comment:
Can ret be an empty array during this return statement?
Do you expect that an empty array will be false, since the object will exist even if it has no elements?
Or, is the intention that this return statement should always be true?

Reply:
I should probably add some JSDoc for it, but the intention is for it to return a combined list of options names for a given `ICommandProfileProperty`.
The expected outcome is an array of strings which results in either `["optionName"]`, or `[undefined, "optionA", "optionB", ...]` which will later get unified (`lodash.union`) with the `specialValues` array.

Reply:
JSDoc was added